corporate/calendar.rst

```diff
@@ -1,27 +1,90 @@
+:nosearch:
+:og:description: This page describes tasks that must be completed on a set schedule.
+
 Calendar of tasks
 =================
 
-.. todo:: finish writing this page
-
+.. vale Google.LyHyphens = NO
 .. vale Google.Passive = NO
-.. vale write-good.Passive = NO
 .. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+This page describes tasks that must be completed on a set schedule.
+
+January
+-------
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Submit Sales & Use Tax Return to the Georgia Department of Revenue
+* `File Annual Registration with the Georgia Secretary of State <https://sos.ga.gov/how-to-guide/how-file-annual-registration>`_
+* Hold quarterly board meeting
```

Comment:
Will there be a board meeting planning page?

Reply:
Added stub page